app/api/v2/handlers/fact_api.py

```diff
@@ -67,9 +68,11 @@ async def get_relationships(self, request: web.Request):
                 raise JsonHttpBadRequest(error_msg)
         return web.json_response(dict(found=resp))
 
-    @aiohttp_apispec.docs(tags=['facts'])
+    @aiohttp_apispec.docs(tags=['facts'],
+                          summary='Create facts',
+                          description='Create facts by criteria provided in the `FactSchema`.')
```

Comment:
This can also be updated to be something like `Create a fact using the format provided in the `FactSchema`.